aiida/orm/groups.py

```diff
@@ -109,7 +111,20 @@ def delete(self, pk: int) -> None:
         self._backend.groups.delete(pk)
 
 
-class Group(entities.Entity['BackendGroup'], entities.EntityExtrasMixin, metaclass=GroupMeta):
+class GroupBase:
+    """A namespace for group related functionality, that is not directly related to its user-facing properties."""
```

Comment:
Still not sure about the "not directly related to its user-facing properties" because, if anything, the extras are exactly intended to be used by users. But it is just a docstring so not that important